### PR TITLE
Fix whack github highlighting

### PR DESCRIPTION
**Pull request by camjc**
Should be using a curly apostrophe anyway, so two birds with one stone here.

### CODE REVIEW

**Comment by taion** [comment]
I don't think we want non-ASCII source

**Comment by camjc** [comment]
No worries, shall I close this?

**Comment by taion** [comment]
Depends how much this syntax highlighting thing bothers you. The HTML entity for an appropriate apostrophe could work.

**Comment by taion** [comment]
Thanks!

File: examples/Overlay.js [hunk 1]
```diff
@@ -113,7 +113,7 @@ const OverlayExample = React.createClass({
           target={ props => findDOMNode(this.refs.target)}
         >
           <ToolTip>
-            I'm placed to the: <strong>{this.state.placement}</strong>
```
Comment (oliviertassinari) [inline examples/Overlay.js]: I believe that we could have solve it this way:
```jsx
{"I'm placed to the: <strong>{this.state.placement}</strong>"}
```
To be confirmed.

Comment (taion) [inline examples/Overlay.js]: Yeah... frankly I'm surprised the GH syntax highlighting fails so badly here. Oh well.

Comment (taion) [inline examples/Overlay.js]: This isn't actually the JSX mode per se – there is a GH JSX mode but it's a bit wonky.

**Pull request merged**